connexion/apps/aiohttp_app.py

```diff
@@ -29,7 +30,8 @@ def create_app(self):
     def get_root_path(self):
         mod = sys.modules.get(self.import_name)
         if mod is not None and hasattr(mod, '__file__'):
-            return os.path.dirname(os.path.abspath(mod.__file__))
+            root_path = os.path.dirname(os.path.abspath(mod.__file__))
+            return pathlib.Path(root_path)
```

Comment:
We can also leverage pathlib to do path operations:
```suggestion
            return pathlib.Path(mod.__file__).resolve().parent
```

See: https://docs.python.org/3/library/pathlib.html#correspondence-to-tools-in-the-os-module